talpid-tunnel-config-client/examples/psk-exchange.rs

```diff
@@ -11,23 +11,26 @@ async fn main() {
         .next()
         .expect("Give tuncfg server IP as first argument")
         .parse()
-        .expect("tuncfg ip argument not a valid IP");
-    let pubkey_string = args
+        .expect("tuncfg ip argument not a valid IPv4");
```

Comment:
If the example should strictly require IPv4 then you also need to change the parsing to parse as `Ipv4Addr` and not `IpAddr`. It's strange if the argument "lol" fails with "Not valid IPv4" but "2001::1" does not.

Reply:
That's a good point. I will change it.